src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -883,6 +975,19 @@ public bool HideAutocompleteMethodOptions
             }
         }
 
+        /// <summary>
+        /// Returns if the current session is Dynamo Revit
+        /// </summary>
+        public bool IsDynamoRevit
+        {
+            get
+            {
+                // HostAnaltyicsInfo is not set when this is invoked??
+                //return this.dynamoViewModel.Model.HostAnalyticsInfo.HostName.Equals("Dynamo Revit");
+                return this.dynamoViewModel.Model.HostName.Equals("Dynamo Revit");
```

Comment:
Model.HostName will be deprecated. Isn't HostAnaltyicsInfo providing the host name? Is that null here?

Reply:
That's exactly right. The HostAnalyticsInfo is null when we need it, however the old Model.HostName has the correct value.

Reply:
Can the HostAnalyticsInfo be null and HostName have the correct host in any case? May be HostAnalyticsInfo is set later in the execution?.

Reply:
Updated as per @mjkkirschner comment - now checks for HostAnalyticsInfo first, if that is `null` goes on to check Model.HostName.